app/mobile-content/exhibits/shared/exhibit.service.ts

```diff
@@ -0,0 +1,126 @@
+import { Injectable } from '@angular/core';
+import { Response } from '@angular/http';
```

Comment:
It should be:
`import { Injectable } from '@angular/core';
import { Response } from '@angular/http';
import { BehaviorSubject } from 'rxjs/Rx';`
(our CG: external imports ordered by package name of the package imported from due to the fact that multiple classes can be imported from the same package)